qe_metrics/libs/database.py

```diff
@@ -127,3 +134,44 @@ class JiraIssues(DB_CONNECTION.Entity):  # type: ignore
         customer_escaped = orm.Required(bool)
         date_created = orm.Required(date)
         last_updated = orm.Required(date)
+        date_record_modified = orm.Required(datetime, default=datetime.now)
+
+        @classmethod
+        def create_update_issues(
+            cls, issues: List[Issue], product: "Database.Products", severity: str, jira_server: str
+        ) -> None:
+            """
+            Create or update JiraIssues items in the database from a list of Jira issues.
+
+            Args:
+                issues (List[Issue]): A list of Jira issues
+                product (Database.Products): A product object
+                severity (str): Severity of the issues
+                jira_server (str): Jira server URL
+
+            Returns:
+                List["Database.JiraIssues"]: A list of JiraIssues objects
+            """
+            issues = [_issue for _issue in issues if _issue.fields.issuetype.name.lower() == "bug"]
```

Comment:
Don't we process all issues that we got from the query? 
The user needs to have a proper query.

Reply:
@myakove I feel some check (either in the PR process or code) has to be there to ensure the additional ` issuetype` (story/task/epic/etc) are not added in the DB. Even if the responsibility lies with the user to provide the query, there should be some validation on the tool side. The check in the code seems to be a simple check instead of adding this in the CI-pipeline .